source/SIL.AppBuilder.Portal/common/public/workflow.ts

```diff
@@ -0,0 +1,138 @@
+import type { RoleId } from './prisma.js';
+
+export enum ActionType {
+  /** Automated Action */
+  Auto = 0,
+  /** User-initiated Action */
+  User
+}
+
+/**
+ * The administrative requirements of the workflow.
+ * Examples:
+ *  - If the flow has `WorkflowAdminLevel.High` it will include extra state to represent the organizational approval process
+ *  - If the flow has `WorkflowAdminLevel.Low` it will not include those states, but there are still some states that require action from an OrgAdmin to complete certain actions
+ *  - If the flow has `WorkflowAdminLevel.None` none of the states or actions for the workflow instance will require an OrgAdmin.
+ * 
+ * Any state or transition can have a list of specified `WorkflowAdminLevel`s. What this means is that those states and transitions will be included in a workflow instance ONLY when the instance's `WorkflowAdminLevel` is in the state's or transition's list.
+ * 
+ * If a state or transition does not specify any `WorkflowAdminLevel` it will be included (provided it passes other conditions not dependent on `WorkflowAdminLevel`).
+*/
+export enum WorkflowAdminLevel {
```

Comment:
I'd still rather this have the word "Verification" or "Required" in it somewhere. As is this makes me think that the workflow has admin privileges, not that it requires admin input.